src/apps/filters/h3.c

```diff
@@ -1632,6 +1632,811 @@ SUBCOMMAND(cellsToMultiPolygon,
+    bool isValid = H3_EXPORT(isValidDirectedEdge)(cell);
+    printf("%s", isValid ? "true\n" : "false\n");
+    return E_SUCCESS;
+}
+
+SUBCOMMAND(getDirectedEdgeOrigin,
+           "Returns the origin cell from the directed edge") {
+    DEFINE_CELL_ARG(cell, cellArg);
+    Arg *args[] = {&getDirectedEdgeOriginArg, &cellArg, &helpArg};
+    PARSE_SUBCOMMAND(argc, argv, args);
+    H3Index out = 0;
+    H3Error err = H3_EXPORT(getDirectedEdgeOrigin)(cell, &out);
+    if (err != E_SUCCESS) {
+        return err;
+    }
+    printf("%" PRIx64 "\n", out);
+    return E_SUCCESS;
+}
+
+SUBCOMMAND(getDirectedEdgeDestination,
+           "Returns the destination cell from the directed edge") {
+    DEFINE_CELL_ARG(cell, cellArg);
+    Arg *args[] = {&getDirectedEdgeDestinationArg, &cellArg, &helpArg};
+    PARSE_SUBCOMMAND(argc, argv, args);
+    H3Index out = 0;
+    H3Error err = H3_EXPORT(getDirectedEdgeDestination)(cell, &out);
+    if (err != E_SUCCESS) {
+        return err;
+    }
+    printf("%" PRIx64 "\n", out);
+    return E_SUCCESS;
+}
+
+SUBCOMMAND(
+    directedEdgeToCells,
+    "Returns the origin, destination pair of cells from the directed edge") {
+    DEFINE_CELL_ARG(cell, cellArg);
+    Arg *args[] = {&directedEdgeToCellsArg, &cellArg, &helpArg};
+    PARSE_SUBCOMMAND(argc, argv, args);
+    H3Index out[2] = {0};
+    H3Error err = H3_EXPORT(directedEdgeToCells)(cell, &out[0]);
+    if (err != E_SUCCESS) {
+        return err;
+    }
+    printf("[%" PRIx64 ", %" PRIx64 "]\n", out[0], out[1]);
+    return E_SUCCESS;
+}
+
+SUBCOMMAND(originToDirectedEdges,
+           "Returns all of the directed edges from the specified origin cell") {
+    DEFINE_CELL_ARG(cell, cellArg);
+    Arg *args[] = {&originToDirectedEdgesArg, &cellArg, &helpArg};
+    PARSE_SUBCOMMAND(argc, argv, args);
+    H3Index out[6] = {0};
+    // This one is pretty loose about the inputs it accepts, so let's validate
+    // for it
+    bool isValid = H3_EXPORT(isValidCell)(cell);
+    if (!isValid) {
+        return E_CELL_INVALID;
+    }
+    H3Error err = H3_EXPORT(originToDirectedEdges)(cell, &out[0]);
+    if (err != E_SUCCESS) {
+        return err;
+    }
+    printf("[");
+    bool hasPrinted = false;
+    for (int i = 0; i < 6; i++) {
+        if (out[i] > 0) {
+            if (hasPrinted) {
+                printf(", ");
+            }
+            printf("%" PRIx64, out[i]);
+            hasPrinted = true;
+        }
+    }
+    printf("]\n");
+    return E_SUCCESS;
+}
+
+SUBCOMMAND(directedEdgeToBoundary,
+           "Provides the coordinates defining the directed edge") {
+    DEFINE_CELL_ARG(cell, cellArg);
+    Arg *args[] = {&directedEdgeToBoundaryArg, &cellArg, &helpArg};
+    PARSE_SUBCOMMAND(argc, argv, args);
+    CellBoundary cb = {0};
+    H3Error err = H3_EXPORT(directedEdgeToBoundary)(cell, &cb);
+    if (err) {
+        return err;
+    }
+    // Using WKT formatting for the output. TODO: Add support for JSON
+    // formatting
```

Comment:
Is it weird that we use `WKT` here but raw `lat,lng` polygons for `cellsToMultiPolygon`?